modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm

```diff
@@ -74,8 +74,8 @@ sub default_options {
         'blastdb_dir'     => $self->o('pipeline_dir').'/blast_db',
         'blastdb_name'    => $self->o('file_basename').'.pep',
 
-        'uniprot_rel_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/reldate.txt',
-        'uniprot_ftp_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/uniprot_#uniprot_source#_#tax_div#.dat.gz',
+        'uniprot_rel_url' => 'https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/reldate.txt',
+        'uniprot_ftp_url' => 'https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/uniprot_#uniprot_source#_#tax_div#.dat.gz',
```

Comment:
I'm curious if this variable`uniprot_ftp_url` should be renamed as well.

What's your opinion @twalsh-ebi ?

Reply:
ignore my comment. Ftp is still part of the link - I believe variable name change wont be needed